Sources/GRPCInteroperabilityTestsImplementation/TestServiceAsyncProvider.swift

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2021, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#if compiler(>=5.5)
+import Foundation
+import GRPC
+import GRPCInteroperabilityTestModels
+import NIO
```

Comment:
nit: use `NIOCore` (`NIO` is an umbrella import for `NIOCore`, `NIOPosix` and `NIOEmbedded`).